crates/ruff/tests/integration_test.rs

```diff
@@ -1502,6 +1555,11 @@ fn check_input_from_argfile() -> Result<()> {
           |
           = help: Remove unused import: `os`
 
+        Suggested fix:
```

Comment:
I would remove the newline (so each violation is a singe block) and drop "Suggested fix" (I think the "help" is a sufficient introduction).

Reply:
Done in 1f07f5f539f3.